tests/e2e/tests/test_redirects.py

```diff
@@ -173,23 +173,23 @@ def test_that_checks_redirect_using_incorrect_query_values(self, base_url):
         }
         response = self._head_request(base_url, params=param)
 
-        assert (requests.codes.not_found == response.status_code,
-                self.response_info_failure_message(base_url, param, response))
+        assert requests.codes.not_found != response.status_code, \
```

Comment:
oops, this was debugging to make sure a failure reported correctly - thanks for catching this (wish we had Travis CI integration already).